packages/cadl-typescript/src/modelUtils.ts

```diff
@@ -289,6 +293,35 @@ function isSchemaProperty(program: Program, property: ModelProperty) {
   return !(headerInfo || queryInfo || pathInfo || statusCodeinfo);
 }
 
+function extractPagedMetadataNested(
+  program: Program,
```

Comment:
I don't think we need to define this func again.
https://github.com/Azure/autorest.typescript/blob/3ee5d2733b8be991c7787937a6fd4ad554f1a3ea/packages/cadl-typescript/src/operationUtil.ts#L160